test/test_manager.py

```diff
@@ -99,7 +99,19 @@ class ManagerConfig(Config):
     reconfigure_screens = False
 
 
-manager_config = pytest.mark.parametrize("manager", [ManagerConfig], indirect=True)
+class MultiScreenManagerConfig(ManagerConfig):
+    screens = [
+        libqtile.config.Screen(
+            bottom=libqtile.bar.Bar(
+                [libqtile.widget.GroupBox()], 20
+            ),
+        )
+        for _ in range(3)
+    ]
+
+
+manager_config = pytest.mark.parametrize("qtile", [ManagerConfig], indirect=True)
+multi_screen_manager_config = pytest.mark.parametrize("qtile", [MultiScreenManagerConfig], indirect=True)
```

Comment:
```suggestion
manager_config = pytest.mark.parametrize("manager", [ManagerConfig], indirect=True)
multi_screen_manager_config = pytest.mark.parametrize("manager", [MultiScreenManagerConfig], indirect=True)
```
This is the cause of all the test failures. The first argument is the name of the fixture that you're parametrizing so it has to be `manager` here.